Makefile

```diff
@@ -80,14 +87,20 @@ $(HP_DATA_CLEAN):
 	cd generator-outputs/household_poverty/ && \
 	$(PYTHON) $(QUIPP_ROOT)/generators/household_poverty/clean.py
 
+# generate the three artificial datasets
+$(ARTIFICIAL_DATA_1) $(ARTIFICIAL_DATA_2) $(ARTIFICIAL_DATA_3)  $(ARTIFICIAL_DATA_4)  $(ARTIFICIAL_DATA_5)  $(ARTIFICIAL_DATA_6)  $(ARTIFICIAL_DATA_7):
+	mkdir -p generator-outputs/artificial/ && \
+	cd generator-outputs/artificial/ && \
+	$(PYTHON) $(QUIPP_ROOT)/generators/artificial/generate.py
+
 
 ##-------------------------------------
 ## Generate synthetic data
 ##-------------------------------------
 
 ## synthesize data - this rule also builds "synth-output/%/data_description.json"
 $(SYNTH_OUTPUTS_CSV) : \
-synth-output/%/synthetic_data_1.csv : run-inputs/%.json $(AE_DEIDENTIFIED_DATA) $(HP_DATA_CLEAN)
+synth-output/%/synthetic_data_1.csv : run-inputs/%.json $(AE_DEIDENTIFIED_DATA) $(HP_DATA_CLEAN) $(ARTIFICIAL_DATA_1) $(ARTIFICIAL_DATA_2) $(ARTIFICIAL_DATA_3) $(ARTIFICIAL_DATA_4) $(ARTIFICIAL_DATA_5) $(ARTIFICIAL_DATA_6) $(ARTIFICIAL_DATA_7)
```

Comment:
These were added too make sure artificial datasets are created at the beginning of the pipeline run. Will need to be moved to allow CI to run when merged to develop but maybe OK to keep them in this PR?

Reply:
I left the artificial ones in as they don't require manual steps. But removed $(AE_DEIDENTIFIED_DATA) $(HP_DATA_CLEAN)

Reply:
Doest the $(AE_DEIDENTIFIED_DATA) require a manual input? I thought it just downloaded a file automatically.